frontend/src/pages/premium.page.tsx

```diff
@@ -198,34 +200,63 @@ const PremiumPromo: NextPage = () => {
+                  heading: l10n.getString("landing-use-cases-shopping"),
+                  content: (
+                    <CarouselContentHero
+                      heroImage={ShoppingHero.src}
+                      heading={l10n.getString(
+                        "landing-use-cases-shopping-hero-heading"
+                      )}
+                      body={
+                        <>
+                          <p>
+                            {l10n.getString(
+                              "landing-use-cases-shopping-hero-content1"
+                            )}
+                          </p>
+                          <p>
+                            {l10n.getString(
+                              "landing-use-cases-shopping-hero-content2"
+                            )}
+                          </p>
+                        </>
+                      }
+                      textFirst={true}
+                    />
                   ),
                   illustration: ShoppingIllustration,
                   id: "use-cases/shopping",
                 },
                 {
-                  color: "orange",
-                  heading: l10n.getString(
-                    "premium-promo-use-cases-social-networks-heading"
+                  alt: l10n.getString(
+                    "landing-use-cases-social-networks-image-alt"
                   ),
-                  content: l10n.getString(
-                    "premium-promo-use-cases-social-networks-body-2"
+                  color: "deep-pink",
+                  heading: l10n.getString("landing-use-cases-social-networks"),
+                  content: (
+                    <CarouselContentTextOnly
+                      heading={l10n.getString(
+                        "landing-use-cases-social-networks"
+                      )}
+                      body={l10n.getString(
+                        "landing-use-cases-social-networks-body"
+                      )}
+                    />
                   ),
                   illustration: SocialNetworksIllustration,
                   id: "use-cases/social-networks",
                 },
                 {
-                  color: "pink",
-                  heading: l10n.getString(
-                    "premium-promo-use-cases-gaming-heading"
-                  ),
-                  content: l10n.getString(
-                    "premium-promo-use-cases-gaming-body-2"
+                  alt: l10n.getString("landing-use-cases-gaming-image-alt"),
+                  color: "orange",
+                  heading: l10n.getString("landing-use-cases-gaming"),
+                  content: (
+                    <CarouselContentTextOnly
+                      heading={l10n.getString("landing-use-cases-gaming")}
+                      body={l10n.getString("landing-use-cases-gaming-body")}
+                    />
                   ),
                   illustration: GamingIllustration,
                   id: "use-cases/gaming",
```

Comment:
I just checked the ticket, and I think it also wanted to have Social Networks and Gaming to be replaced by the homepage's On the Go and Signups, respectively.